### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by srueg**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Update the ./CHANGELOG.md.
- [x] Link this PR to related issues.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@ SHELL := bash
...
 DOCKER_ARGS  ?= run --rm --user "$$(id -u)" -v "$${PWD}:/component" --workdir /component
 
-JSONNET_FILES   ?= $(shell find . -type f -name '*.*jsonnet' -or -name '*.libsonnet')
+JSONNET_FILES   ?= $(shell find component/ -type f -name '*.*jsonnet' -or -name '*.libsonnet')
 JSONNETFMT_ARGS ?= --in-place --pad-arrays
 JSONNET_IMAGE   ?= docker.io/bitnami/jsonnet:latest
 JSONNET_DOCKER  ?= $(DOCKER_CMD) $(DOCKER_ARGS) --entrypoint=jsonnetfmt $(JSONNET_IMAGE)
 
-YAML_FILES      ?= $(shell find . -type f -name '*.yaml' -or -name '*.yml')
+YAML_FILES      ?= $(shell find class/ -type f -name '*.yaml' -or -name '*.yml')
```
Comment (corvus-ch) [inline Makefile:16]: This has the risk of being missed when adding new files else where. For https://github.com/appuio/component-openshift4-monitoring/pull/10 I solved this by excluding the ones I did not care.

This is the fail save approach.  Consider to do the same.

**Review by corvus-ch — CHANGES_REQUESTED** [review]
What is the goal of this component?

Thanos is a tool that can be used in many different ways and can solve different problems. Please use the documentation to explain the results of this component, what setup it creates and what problems it does address.

**Review by corvus-ch — APPROVED** [review]
LGTM

**Comment by corvus-ch** [comment]
Excluding `vendor` from linting might be a good thing in general.  Can you (@srueg) create a PR on the component template?

**Review by simu — APPROVED** [review]
LGTM apart from the typo in the docs

**Pull request merged**